### DISCUSSION TITLE
Latest version of Astroid compatible with Joomla 3

### DISCUSSION THREAD

**Discussion by iorbita** [templaza/astroid-framework#758] [Q&A]
Hi,
I have to upgrade a complex site from Joomla 3 to Joomla 4, I hesitate to start from scratch, but it's a lot of work. I think I'll try an upgrade first and see how it goes... 

The current framework and template (Joomlabamboo) are not compatible with Joomla 4, so I'd like to install Astroid with a template and then migrate to Joomla 4.

What is the latest version of Astroid compatible with Joomla 3?

Many thanks,
Lorenzo

**Comment by sonvnn** [comment]
Astroid 2 is compatible with Joomla 3. You can download and check it here https://github.com/templaza/astroid-framework/tree/v2

**Reply by iorbita** [reply]
> Astroid 2 is compatible with Joomla 3. You can download and check it here https://github.com/templaza/astroid-framework/tree/v2

thanks!
